protocol-doc.md

```diff
@@ -203,6 +203,14 @@ represented by a space), and the rest of the string being their username.
 
 > Indicates that the game has started.
 
+`|win|USER`
+
+> `USER` has won the battle.
+
+`|tie`
```

Comment:
`/forcetie` is still a command Leaders and Admins can use, even if there is no in-game way for ties to happen. Also, previous gens might have ties.

Documenting battles is really big and I'm still figuring out where to put it into the document organizationally.